packages/webapp/src/app/layout.tsx

```diff
@@ -1,21 +1,17 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import './globals.css';
+import { Inter } from 'next/font/google';
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ['latin'] });
 
 export const metadata = {
   title: 'Create Next App',
   description: 'Generated by create next app',
-}
+};
 
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode
-}) {
+export default function RootLayout({ children }: { children: React.ReactNode }) {
```

Comment:
Q: what does this do?
I wanted to create a parent component for all the pages. For example the ThemeProvider would live there.
Can this `RootLayout` thing do that?

Reply:
If yes, I'd like to push the `AppHeader` into the RootLayout too.

Reply:
TBH I don't really know for sure. I think it is the main page layout for all pages in the app. Check the Next.js docs :-)